.circleci/config.yml

```diff
@@ -144,7 +144,6 @@ jobs:
       - *yarn_install
 
       - run: yarn bazel build src/... --build_tag_filters=-docs-package
```

Comment:
should we rename the job above to `bazel_build` instead?

Reply:
Sure!